Assets/Scripts/System/Ecs7GenerationSystem.cs

```diff
@@ -351,6 +351,9 @@ private async Task<PropertyInfo> GatherPropertyInfo(DclComponent component, DclC
             case DclComponent.DclComponentProperty.PropertyType.String:
                 value = $"\"{property.GetConcrete<string>().FixedValue}\"";
                 break;
+            case DclComponent.DclComponentProperty.PropertyType.Color:
+                value = $"\"{ColorUtility.ToHtmlStringRGBA(property.GetConcrete<Color>().FixedValue)}\"";
```

Comment:
This needs to be fixed. Currently, it will generate a string 
![grafik](https://github.com/thehubdao/dcl-edit/assets/11379989/ee617f76-6004-4094-82c3-64cdbb76d262)
But it should do a Color4.create call.
![grafik](https://github.com/thehubdao/dcl-edit/assets/11379989/c81ecd64-fef7-4548-be44-a4a9f1f84776)